src/ServiceControlInstaller.PowerShell/Cmdlets/ServiceControlInstances/InvokeServiceControlInstanceUpgrade.cs

```diff
@@ -27,6 +27,9 @@ public class InvokeServiceControlInstanceUpgrade : PSCmdlet
         [ValidateTimeSpanRange(MinimumHours = 240, MaximumHours = 1080)] //10 to 45 days
         public TimeSpan? ErrorRetentionPeriod { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Do not automatically create new queues")]
```

Comment:
Would we ever create the queues when upgrading an instance?

Reply:
Yes. If we added a satellite or something. We enable installers so we'll try and create queues.